src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -803,6 +791,23 @@ private function checkPermissions(array $pageVoter)
         }
     }
 
+    /**
+     * @param string $pageVoter
+     *
+     * @return JsonResponse
+     */
+    private function checkPermission(string $pageVoter)
```

Comment:
This won't work in PHP 5.x
```suggestion
    private function checkPermission($pageVoter)
```

Reply:
It's strange that it was not found by PHP syntax checker 🤔 or is that a check we added from 1.7.7 ?

Reply:
It's a Github action I think?

Reply:
That's not a syntax but a runtime error, that's why it wasn't found by `php -l`